### PR TITLE
Fix bug 1130383 - Get rid of test-utils.

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

**Comment by robhudson** [comment]
Don't forget settings.py... settings.INSTALLED apps and settings.TEST_RUNNER.

**Comment by robhudson** [comment]
r+ when tests pass.

**Comment by ubernostrum** [comment]
Should have sent... a poet...

jodileansm

File: manage.py [hunk 1]
```diff
@@ -24,15 +24,24 @@ def path(*parts):
...
+        try:
+            import settings_local  # noqa
+        except ImportError:
+            # or use the plain settings.py module
+            settings_mod = 'settings'
+        else:
+            settings_mod = 'settings_local'
+
+    # override the env var with what we want
+    os.environ['DJANGO_SETTINGS_MODULE'] = settings_mod
```
Comment (robhudson) [inline manage.py]: Are you happy with this?

File: settings_travis.py [hunk 2]
```diff
@@ -1,10 +1,4 @@
-from settings import *
+from settings_test import *
 
-CELERY_ALWAYS_EAGER = True
-ASYNC_SIGNALS = False
-ES_URLS = ['localhost:9200']
-ES_INDEX_PREFIX = 'mdn'
-ES_INDEXES = {'default': 'main_index'}
-ES_INDEXING_TIMEOUT = 30
```
Comment (robhudson) [inline settings_travis.py]: The `ES_LIVE_INDEX` is now different. That ok?

File: tox.ini [hunk 3]
```diff
@@ -7,7 +7,7 @@ deps=
     -rrequirements/compiled.txt
     -rrequirements/dev.txt
 commands =
-    coverage run manage.py test --noinput -v2 kuma
+    coverage run manage.py test --noinput -v2 {posargs:kuma}
     coverage report
 setenv =
     FORCE_DB = 1
```
Comment (robhudson) [inline tox.ini]: I think this `FORCE_DB` can go away now. That's a test-utils only thing iirc. The nose test runner has something equivalent but it's different... you have to tell it to reuse the db explicitly.

**Comment by robhudson** [comment]
Back to you with questions and one minor fix.

File: manage.py (hunk 1 above)
Comment (jezdez) [inline manage.py]: Hahaha, "happy". No

File: settings_travis.py (hunk 2 above)
Comment (jezdez) [inline settings_travis.py]: We want to test against ES on Travis, yeah

File: tox.ini (hunk 3 above)
Comment (jezdez) [inline tox.ini]: Not the nose test runner from 2010 or 2011 we're using :'(

Comment (jezdez) [inline tox.ini]: And yeah I'll remove FORCE_DB

Comment (jezdez) [inline tox.ini]: Or should I say FARCE_DB

Comment (jezdez) [inline tox.ini]: Sigh

**Comment by groovecoder** [comment]
To run the tests in my vm, I had to copy the `DATABASES` setting from `settings_local.py` into `settings_test.py` now that we only load one or the other in `manage.py`? Is that the right way to run locally, or do we need to update test instructions somehow?

**Comment by groovecoder** [comment]
ping @jezdez - what do you think? Should we copy the `DATABASES` setting into `settings_test.py` so dev environments can run the tests out-of-the-box?

**Comment by jezdez** [comment]
@groovecoder No, we should update the `settings.py` to have the database setting that we expect for local development including testing, I don't understand why we even have an "empty" `DATABASES` setting in there that won't work anyway. That means that the Travis settings need an overridden `DATABASES` setting using the root user following http://docs.travis-ci.com/user/database-setup/#MySQL

**Comment by groovecoder** [comment]
Makes sense. So you'll fill in the `DATABASES` setting in `settings.py` as part of this PR? Or is that a step for another PR?

**Comment by jezdez** [comment]
@groovecoder Part of this PR

**Comment by groovecoder** [comment]
I checked this out and it works locally! Just needs a rebase and then it's good to merge?

**Comment by jezdez** [comment]
@groovecoder Yeah, IMO

**Pull request merged**